config/routes/overrides.rb

```diff
@@ -1,6 +1,21 @@
+class WhitelistConstraint
+  def initialize
+    @path = /^\/api\/v2\/organizations\/\S+\/parameters/
```

Comment:
I was looking at things from the perspective that 'WhitelistConstraint' is quite generic and implies it could contain all of the whitelist rules of which this is just one of them.